### DISCUSSION TITLE
What happened to rebinding the M4 hotkey?

### DISCUSSION THREAD

**Discussion by Valour549** [seerge/g-helper#3212] [Q&A]
In this [thread](https://github.com/seerge/g-helper/discussions/945) you mentioned that the M4 key and Fn+F5 produced the same code, so it wasn't possible to differentiate them.

However I think that has been fixed by ASUS. Now Fn+F5 changes the mode for both AC and G-Helper as expected, but in AC I was able to rebind M4 to an exe you [helped with](https://github.com/seerge/g-helper/discussions/3108) as shown below, with no conflict whatsoever. It'll be great to allow M4 to be rebound once again.

![image](https://github.com/user-attachments/assets/ca0733ae-3876-4528-a918-5c27ee7e3caf)

**Comment by seerge** [comment]
@Valour549 hello. 

AC can rebind some keys on hardware level, while G-Helper listens to existing key-codes hotkeys emit and processes corresponding action (so whole "binding" is purely on software level). I'm keen to current solution as it is much less invasive and can't screw up keyboard keys actions :)
